src/components/Summary/DocumentsDelivery.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <section id="document-delivery-section">
+    <header>
+      <h2>X. Alteration Documents Delivery</h2>
+    </header>
+    <div class="pt-4">Copies of the alteration documents will be sent to the following email address listed below.</div>
+    <div class="pl-8 pt-8 pr-8">
+      <v-layout row pb-6>
+        <v-flex xs3 md2 class="registered">
```

Comment:
v-layout and v-flex are deprecated.
Use v-container, v-row and v-col instead.

https://www.google.com/search?q=vuetify+v-layout+deprecated
https://vuetifyjs.com/en/getting-started/upgrade-guide/#grid
https://vuetifyjs.com/en/components/grids/#v-row

Reply:
(Flexbox helper classes are still OK I think.)

https://vuetifyjs.com/en/styles/flex/#flex

Reply:
I've made this change